src/main/resources/templates/fragments/results.html

```diff
@@ -3,6 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <title>Learning Outcomes Management System</title>
+    <link href="../../static/css/bootstrap.css" rel="stylesheet" th:href="@{/css/bootstrap.css}"/>
```

Comment:
Dont think you need to add these link to results,html since it's populated into index.html with the link calls already

Reply:
^I believe that's true as well, @SupriyaGadigone can you test this so that we don't have unnecessary links in results?